CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 ## master
 
+## 1.3.2 (2020-08-10)
```

Comment:
Please add 1.3.1 and 1.3.2 changes under ## master

Reply:
Removed in-midst-versions, left both changes within the master branch